spec/Appendix C -- Tooling Conventions.md

```diff
@@ -0,0 +1,103 @@
+# C. Tooling Conventions
+
+This specification document formalizes naming and conventions that may be used by
+GraphQL tooling and third party libraries.
+
+
+## Field Coordinates
+
+FieldCoordinate : TypeName `.` FieldName
+
+A field coordinate may be used to uniquely identify a field in a GraphQL schema.
+It is encoded as a string containing a dot seperated parent type and field pair.
+
+**Examples**
```

Comment:
Do you mind to write example for array?

Reply:
Could you elaborate? I think this should look the same for nullability/arrays etc - but I will add another example for clarity and avoidance of doubt. Is there something specific you were looking for?

thanks!

(What we do for arrays and nullability will get more complicated for serialization - still working on this section)

Reply:
It seemed to me that for arrays field coordinate should include an index. I might be wrong.